README.md

```diff
@@ -1,6 +1,6 @@
-This Readme covers release 16.* of the LORIS Imaging Insertion Pipeline for Ubuntu or CentOS systems
+This Readme covers release 16.1 of the LORIS Imaging Insertion Pipeline for Ubuntu or CentOS systems
 
-This repo accompanies the [LORIS neuroimaging data platform main repo](https://github.com/aces/Loris)</b>, release 16.0.*.<br>
+This repo accompanies the [LORIS neuroimaging data platform main repo](https://github.com/aces/Loris/releases)</b>, release 16.1.*.<br>
 For documentation and detailed setup information, please see the [LORIS wiki](https://github.com/aces/Loris/wiki/Imaging-Database)</b>.
 
```

Comment:
@christinerogers 
I would add a small line saying to follow this link, section 1, in particular for CentOS installations